rudder-core/src/main/scala/com/normation/rudder/services/policies/DeploymentService.scala

```diff
@@ -439,10 +439,13 @@ trait DeploymentService_buildNodeConfigurations extends DeploymentService with L
     , globalSystemVariables: Map[String, Variable]
   ): Map[NodeId, InterpolationContext] = {
 
+    println(s"**** AGAIN ${allNodeInfos.size} nodes in allNodeInfos, check exists:\n *** ${List("06da3556-5204-4bd7-b3b0-fa5e7bcfbbea","009e0af1-1a3c-4ee0-a91d-10b68a008f5d", "073239d7-c633-4b60-8477-57131839fd34").map( k => (k, allNodeInfos.isDefinedAt(NodeId(k)))).mkString("\n *** ")}")
```

Comment:
It was debug. The nodes in the list had specific interesting properties, like "no policy server", "policy server id not a known node", "policy server a known node (which don't know it is a policy server)"